lib/peatio/securities/key_generator.rb

```diff
@@ -0,0 +1,18 @@
+module Peatio::Securities
+  class KeyGenerator
+
+    def generate!
+      Dir.mkdir('keys') unless File.exists?('keys')
+      OpenSSL::PKey::RSA.generate(2048).yield_self do |p|
+        { public:  Base64.urlsafe_encode64(p.public_key.to_pem), private: Base64.urlsafe_encode64(p.to_pem) }
+          write('keys/rsa-key.prv', p)
+          write('keys/rsa-key.pub', p.public_key)
+      end
+      puts 'Successfully generated','','Keys have been written to the file'
```

Comment:
Lets change the logic a bit. By default command should generate and output the keypair.
For executing write to the file user should add an option ```--save```